src/screens/token/components/account-settings/account-settings.tsx

```diff
@@ -55,8 +53,9 @@ export class AccountSettingsComponent extends React.Component<IProps & IExternal
         };
     }
 
-    public revealPrivateKey = async () => {
-        const password = await this.passwordModal.requestPassword();
+    public async revealPrivateKey() {
+        // TODO: not working because of modal over modal
```

Comment:
why is there a todo here? - this is not working?

Reply:
This is not working,  I am going to fix this in another task